### DISCUSSION TITLE
Default border color

### DISCUSSION THREAD

**Discussion by socheth** [twbs/bootstrap#31952] [General]
why default border color of class .form-control (#ced4da) is deferent from class .border (#dee2e6) ?
It should be use same color.

**Comment by mdo** [comment]
Good question! They could probably be synced up. Without looking at the Git history, I'm unsure when we made that change for them to be different. I do know we use some other border colors though as well—like `rgba()` values for easier blending with backgrounds. I mention that as not every border will be the same.
